function-reference/version-functions/text.xml

```diff
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd really like @mgorny's review given he wrote https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#version-manipulation-and-comparison-commands.